.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -25,7 +25,13 @@ Share any alternative solutions or features considered.
 **How does this impact you?**  
 Explain the impact to your work or process, e.g., We cannot do [...].
 
-**Are there any deadlines?**  
+**What are the customer data privacy implications for this request?**
+Will (or could) sensitive customer data be exposed as part of this feature? Please test any suggested commands yourself and list any tokens, keys, PII, or other sensitive information that is at risk of being exported or shared via this feature.
+
+**If any sensitive customer data might be exported or shared via this feature, have you spoken with Hashicorp legal about it?**
```

Comment:
This form is for those inside and outside HashiCorp, so let's leave this question out.